Enterprise-CMCS/macpro-mdct-qmr
# Streamline oms

### Description

Certain measures optional measure stratification (OMS) have been changed from showing all of the qualifiers and categories under them to only showing the total categories & qualifiers. 

25 measures were updated to showing certain categories & qualifiers as part of the OMS:

**Adult**
AMR-AD, HBD-AD, IET-AD, OUD-AD 

**Child** 
AMB-CH, APM-CH, APP-CH, CIS-CH, DEV-CH, OEV-CH, TFL-CH, WCC-CH, WCV-CH

**Health Home** 
AIF-HH, AMB-HH, CBP-HH, CDF-HH, FUA-HH, FUH-HH, FUM-HH, IET-HH, IU-HH, OUD-HH, PQI92-HH

5 had them removed completely and will only show a text saying "_CMS is not collecting stratified data for this measure for FFY 2023 Core Set reporting._" under the OMS section: 
CPU-AD, FVA-AD, MSC-AD, PCR-AD, PCR-HH

### Related ticket(s)


MDCT-2443

---
### How to test

1) Sign into QMR, preferably user: [stateuserWA@test.com](mailto:stateuserDC@test.com) as it has Health Home.
2) Select any measure listed above.
3) Click the first radio button for the first 3 questions, that will enable the Performance Measure for the N/D/R.
4) Fill in values for N/D/R and that will enable the Optional Measure Stratification
5) Select any check box for OMS, make sure the N/D/R qualifier/categories displayed matches what is listed in MDCT-2443 for the selected measure.

Note: Ben created a cleaned readme called StorySummary_MDCT-2443.md that simplifies what measure has what fields. It is attached on the ticket.

6) Enter value in the OMS and validate to make sure there's no validation error.


### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
